plugins/inputs/prometheus/sample.conf

```diff
@@ -125,3 +125,13 @@
 
   ## Use TLS but skip chain & host verification
   # insecure_skip_verify = false
+
+  ## Control pod scraping based on pod namespace annotations
+  #
+  # Pass and drop here act like tagpass and tagdrop, but instead
+  # of filtering metrics they filters pod candidates for scraping
+  #[inputs.prometheus.namespace_annotation_pass]
+  # annotation_key = ["value1", "value2"]
+  #[inputs.prometheus.namespace_annotation_drop]
+  #
```

Comment:
While the default value for both of these is not set, can you add an example like to the annotation_drop like you have for pass?

Reply:
done